### ISSUE TITLE
It's broken

### ISSUE THREAD

**Issue by onbh943** [forscht/Piped#1]
The download keeps failing, either at the beginning or at 100%. 

Purely FYI

**Comment by jacobvd0** [comment]
Were you trying to upload to piped.cc? Pretty sure it doesn't work anymore but selfhosting it still works

**Comment by onbh943** [comment]
Yes, uploading to piped.cc so I guess it was just replaced by ddrive

Thanks for pointing this out

**Comment by jacobvd0** [comment]
No problem, I've got an instance at share.jacob9086.xyz if you wanted a public instance to use at all.

------- Original Message -------
